.golangci.yml

```diff
@@ -1,8 +1,9 @@
+run:
+  timeout: 5m
+
 linters-settings:
   gofmt:
     simplify: false
-  govet:
-    check-shadowing: false
```

Comment:
The config structure changed (moved to `settings.shadow`) and it's also the default value.